Frameworks/CoreFoundation/Preferences.subproj/CFPreferences.c

```diff
@@ -882,11 +882,7 @@ static Boolean synchronizeApplicationDomain(CFTypeRef context, void *domain) {
     CFDataRef data = CFPropertyListCreateData(__CFPreferencesAllocator(), (CFPropertyListRef)domain, kCFPropertyListXMLFormat_v1_0, 0, NULL);
     Boolean success = false;
     if (data) {
-#if DEPLOYMENT_TARGET_WINDOWS
-        success = _CFWriteBytesToFileAsync((CFURLRef)context, CFDataGetBytePtr(data), CFDataGetLength(data));
```

Comment:
Please, no! We deliberately added async because we cannot block the UI thread.  What we may want to do though is to wait for the IO operation to complete in case the app is exiting.